main.py

```diff
@@ -0,0 +1,136 @@
+# pylint: disable=superfluous-parens
+import os
+from typing import Dict, Tuple
+from uuid import UUID
+
+import simplejson as json
```

Comment:
Any specific reason for `simplejson`?

Reply:
Only that it's used in Runner. I did briefly look at rapidjson and others and can't see a reason not to use them, but equally I saw nothing that compelled me to change either.

Reply:
I think we should use another faster parser if we think we might benefit, but we don't do much JSON processing so could use built in. Runner uses simplejson for its functionality with serializing/deserialising Decimals etc which is not needed here.

Reply:
What were you thinking - rapidjson or https://pypi.org/project/ujson/?

Reply:
Do we think we will see much benefit?. I'm not sure if we will. ujson is pretty well established and used in libs like FastAPI.

Reply:
Maybe we should use the built in json handling rather than bring in a dependency - https://cloud.google.com/functions/docs/bestpractices/tips#use_dependencies_wisely

Reply:
Doc does indicate it would only be the initial cold start that might be affect given it is at global scope. But I think maybe just use built-in until we identify a reason to look at it.

Reply:
Happy to stick with stdlib unless performance issues arise

Reply:
Done